### ISSUE TITLE
CONSOLE_CAPTURE_PORT error on ilo5

### ISSUE THREAD

**Issue by krisbudhram** [seveas/python-hpilo#179]
With the release of 4.2, we have the following failure when using mod_global_settings

    hpilo.IloFeatureNotSupported: Feature not supported - CONSOLE_CAPTURE_PORT

This failure does not occur with 4.1, and seems to be the result of commit d72f7d340dd283061dc09f8b67f44d4715c57e16

Full trace
```
# hpilo_cli -P local localhost mod_global_settings https_port=443
Traceback (most recent call last):
  File "/bin/hpilo_cli", line 334, in <module>
    main()
  File "/bin/hpilo_cli", line 214, in main
    results = [getattr(ilo, method)(**params)]
  File "/usr/lib/python2.7/site-packages/hpilo.py", line 1464, in mod_global_settings
    return self._control_tag('RIB_INFO', 'MOD_GLOBAL_SETTINGS', elements=elements)
  File "/usr/lib/python2.7/site-packages/hpilo.py", line 734, in _control_tag
    header, message = self._request(root)
  File "/usr/lib/python2.7/site-packages/hpilo.py", line 253, in _request
    message = self._parse_message(data[:pos])
  File "/usr/lib/python2.7/site-packages/hpilo.py", line 588, in _parse_message
    raise subclass(message, status)
hpilo.IloFeatureNotSupported: Feature not supported - CONSOLE_CAPTURE_PORT
```

This failure occurs on ilo5 fw versions 1.15 and 1.20. The ilo advanced license is installed.

We have not tested on ilo4.

**Comment by seveas** [comment]
Yikes!

Thanks for reporting, will release 4.2.1 in a bit.

**Comment by seveas** [comment]
Fixed in ed1183a / 4.2.1

**Comment by krisbudhram** [comment]
Confirmed as working.
